ha/const.py

```diff
@@ -101,3 +106,20 @@
 CONTROLLER_FAILED="Failed"
 CONTROLLER_SUCCESS="Success"
 CONTROLLER_INPROGRESS="InProgress"
+NO_FAILCOUNT = "No failcounts"
+RETRY_COUNT = 2
```

Comment:
RETRY_COUNT of what? Can we rename it better?

Reply:
I think this RETRY_COUNT can be reused for other functionalities too where we need to keep consistent retries on the checks

Reply:
sure